lodash-deep/lodash-deep.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for lodash-deep v 2.0
+
+// Project: https://github.com/marklagendijk/lodash-deep
+// Definitions by: Remo uzust01 <https://github.com/uzust01/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+
+declare module "lodash-deep" {
```

Comment:
This library is an UMD module. Instead you'll need to extend the `lodash` types. You'll find documentation about this on the [library structures page of the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#umd) and on the [template module-plugin.d.ts](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-plugin-d-ts.html).

The types should end like this:

```ts
import * as _ from 'lodash';

declare module 'lodash' {
    interface LoDashStatic { // lodash uses this interface for augmentation
        deepMapValues(
            object: any,
            callback: any,
            propertyPath?: any
        ): any;
    }
}

export function deepMapValues(
    object: any,
    callback: any,
    propertyPath?: any
): any;
```